config/boards/licheepi-4a.wip

```diff
@@ -0,0 +1,17 @@
+# RISC-V LicheePi 4A
+BOARD_NAME="LicheePi 4A"
+BOARDFAMILY="thead"
+BOARD_MAINTAINER=""
+KERNEL_TARGET="legacy"
+BOOT_FDT_FILE="thead/light-lpi4a.dtb"
+SRC_EXTLINUX="yes"
+SRC_CMDLINE="console=ttyS0,115200 rootwait rw earlycon clk_ignore_unused loglevel=7 eth=$ethaddr rootrwoptions=rw,noatime rootrwreset=yes"
```

Comment:
!blame @chainsx: `eth=$ethaddr` expands into `eth=` as `$ethaddr` is seemingly not defined in this scope